app/controllers/v0/forms_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module V0
+  class FormsController < ApplicationController
+    include ActionView::Helpers::SanitizeHelper
+
+    skip_before_action :authenticate
+
+    def index
+      response = Forms::Client.new(query).get_all
```

Comment:
this would typically be `query_params` or just `params`. You're bogged down a bit by your two helper methods